include/openenclave/bits/evidence.h

```diff
@@ -166,7 +166,16 @@ typedef enum _oe_policy_type
      *
      * The policy will be in the form of `oe_datetime_t`.
      */
-    OE_POLICY_ENDORSEMENTS_TIME = 1
+    OE_POLICY_ENDORSEMENTS_TIME = 1,
+
+    /**
+     * Enforces a baseline of endorsements that is defined and used
+     * by third party endorsements info caching service.
+     *
+     * The policy can only be specified once, only the first policy
+     * will be used when multiple policies in this type are given.
```

Comment:
```suggestion
     * will be used when multiple policies of this type are given.
```
However, I think this is bug-prone, since it just encourages bad code to go undetected.
Instead, if you only use the first policy of this type, then to reduce the risk of bugs in the caller, this call should fail if the caller tries to pass multiple of them rather than silently ignoring them.